src/main/java/uk/gov/di/ipv/cri/common/library/domain/personidentity/Address.java

```diff
@@ -153,14 +153,23 @@ public void setValidUntil(LocalDate validUntil) {
 
     @JsonIgnore
     public AddressType getAddressType() {
-        if (Objects.nonNull(this.getValidUntil()) && isPastDate(this.getValidUntil())) {
-            return AddressType.PREVIOUS;
+
+        if (Objects.isNull(this.getValidUntil())) {
+            if (Objects.isNull(this.getValidFrom())) {
```

Comment:
This can be refactored to use a single `if` statement:
```
if ( !validUntilSpecified && ( !validFromSpecified || isPastDateOrToday(this.getValidFrom()) ) ) {
    return AddressType.CURRENT;
}
```